src/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixer.php

```diff
@@ -65,6 +63,15 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
                 if (!$tokens[$possibleDefinitionIndex]->isGivenKind(T_FUNCTION)) {
                     $this->fixFunctionCall($tokens, $index);
                 }
+            } elseif ($tokens[$lastTokenIndex]->equalsAny($braceTypes)) {
+                $type = Tokens::detectBlockType($tokens[$lastTokenIndex]);
+                switch ($type['type']) {
+                    case Tokens::BLOCK_TYPE_ARRAY_INDEX_CURLY_BRACE:
+                    case Tokens::BLOCK_TYPE_DYNAMIC_VAR_BRACE:
+                    case Tokens::BLOCK_TYPE_INDEX_SQUARE_BRACE:
+                    case Tokens::BLOCK_TYPE_PARENTHESIS_BRACE:
+                        $this->fixFunctionCall($tokens, $index);
```

Comment:
swich with only one case is just an if statement. replace `case` with `&&` ;)
remember that consecutive `case` statements without body (or break statement) may be misleading